test/db/cmd/cmd_help

```diff
@@ -47,34 +47,25 @@ RUN
 FILE=malloc://1024
 CMDS=<<EOF
 ?v
-?=
-??
+# ?=
```

Comment:
So what do we do with those? Remove them maybe? They aren't valid commands in the rzshell anyways, since require additional argument, I suppose?

Reply:
Don't forget these